certified-connectors/Asite (Canada)/Readme.md

```diff
@@ -0,0 +1,35 @@
+# About
+
+Asite is an open construction platform that enables organizations with comprehensive range of solutions connect dispersed teams across the lifecycle of capital assets by collaborate, plan, design, and build with seamless information sharing across the entire supply chain which helps capital project owners stay at the forefront of innovation, maintaining a golden thread of information.
```

Comment:
Please update readme file as per the template in this link. 
https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
@vmanoharas I have changed as per this template link.